.github/workflows/update-mds.yml

```diff
@@ -0,0 +1,45 @@
+name: Update README.md
+
+on:
+  pull_request_target:
+    branches:
+      - main
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          ref: ${{github.event.pull_request.head.ref}}
+          repository: ${{github.event.pull_request.head.repo.full_name}}
```

Comment:
What does this setting do? The default is `${{ github.repository }}`, which should be equivalent?

Reply:
The pulled repository didn't have a remote until I configured this option.